Plandomizer.py

```diff
@@ -533,10 +535,33 @@ def fill_bosses(self, world, prize_locs, prizepool):
 
 
     def fill(self, window, worlds, location_pools, item_pools):
+        """Fills the world with restrictions defined in a plandomizer JSON file.
+
+        :param window:
+        :param worlds: A list of the world objects that define the rules of each game world.
+        :param location_pools: A list containing all of the location pools.
+            0: Shop Locations
+            1: Song Locations
+            2: Fill locations
+        :param item_pools: A list containing all of the item pools.
+            0: Shop Items
+            1: Dungeon Items
+            2: Songs
+            3: Progression Items
+            4: Priority Items
+            5: The rest of the Item pool
+        """
         world = worlds[self.id]
         locations = {}
         if self.locations:
             locations = {loc: self.locations[loc] for loc in random.sample(self.locations.keys(), len(self.locations))}
+        if world.open_fountain:
```

Comment:
Please fix: `open_fountain` should be `zora_fountain == 'open'.  
Please also check first that `'Bottle with Letter'` exists in the pool before replacing it and adding another bottle? That seems to throw an error for me.

Reply:
Whoops. I was testing with a 5.1 plando and forgot that setting had changed. It doesn't seem like this bit of logic is required at all when using the proper settings in Dev, so I guess I'll remove it.

Reply:
This does annoy me that plandos aren't necessarily forwards-compatible between release versions. If another option has had this happen that I've forgotten about, or another options changes like this in the future, I might be tempted to try and force them to read the old option value and make sure the new one is set properly. I don't really want to mess with it now if open_fountain/zora_fountain is the only one that has had this issue though.

Reply:
Same with general settings, too. Renaming options or tricks breaks compatibility, and often in a silent way. Maybe we need some sort of upgrader to convert them. :p